lessons/lesson-template.md

```diff
@@ -0,0 +1,74 @@
+- title: 
+- tags: 
+
```

Comment:
If we do, I think we can extract the title from the filename and the tags from learning goals.